src/mz/src/api.rs

```diff
@@ -153,7 +153,7 @@ pub async fn disable_region_environment(
     valid_profile: &ValidProfile<'_>,
 ) -> Result<(), reqwest::Error> {
     Retry::default()
-        .max_tries(10)
+        .max_tries(100)
```

Comment:
As an alternative, if we know the duration, `max_duration(..)` could be more efficient.